tests/integration/telemetry/stats/prometheus/nullvm/accesslogs_test.go

```diff
@@ -69,13 +99,30 @@ spec:
   - providers:
     - name: envoy
     disabled: %v
-`, !expectLogs)
+`, !enableLogs)
+	t.ConfigIstio().YAML(common.GetAppNamespace().Name(), config).ApplyOrFail(t)
+}
+
+func deleteTelemetryResource(t framework.TestContext, enableLogs bool) {
```

Comment:
I'm confused. This looks exactly the same as `applyTelemetryResource`. Shouldn't this end with a `DeleteOrFail` ?